docs/guides/depositor-bot.md

```diff
@@ -0,0 +1,104 @@
+# Lido depositor bot
+
+## Introduction
+
+Depositor bot is a part of [Deposit Security Module](link to dsm)..
+
+The Depositor Bot obtains signed deposit messages from Council Daemons. Once a sufficient number of messages is
+collected to constitute a quorum, the bot proceeds to initiate a deposit into the designated staking module. This
+deposit is executed using
+the [depositBufferedEther](https://docs.lido.fi/contracts/deposit-security-module#depositbufferedether) function within
+the [DepositSecurityModule](https://docs.lido.fi/contracts/deposit-security-module) smart contract.
+
+## Requirements
+
+### Hardware
+
+- 1-core CPU
+- 2GB RAM
+
+### Nodes
```

Comment:
Does it really need nodes, or would RPC services be enough? :thinking:

Reply:
Changed